src/containers/changeAccountEmail/index.tsx

```diff
@@ -0,0 +1,112 @@
+import React, { useState } from 'react';
+import { Helmet } from 'react-helmet';
+import { Alert, Button, Form, Input, Typography } from 'antd';
+import { ContentContainer } from '../../components';
+import ConfirmationMessage from '../../components/ConfirmationMessage';
+import { Link as RouterLink } from 'react-router-dom';
+import ProtectedApiClient from '../../api/protectedApiClient';
+import styled from 'styled-components';
+
+const { Title, Link } = Typography;
+
+const UpdateButton = styled(Button)`
+  margin-top: 20px;
+`;
+
+const ChangeAccountEmail: React.FC = () => {
+  const [loading, setIsLoading] = useState(false);
+  const [error, setError] = useState(false);
+  const [emailChanged, setEmailChanged] = useState(false);
+
+  const onFinishChangeEmail = async (values: any) => {
+    try {
+      setError(false);
+      setIsLoading(true);
+      await ProtectedApiClient.changeAccountEmail(values);
+      setEmailChanged(true);
+    } catch (err) {
+      setError(true);
+    } finally {
+      setIsLoading(false);
+    }
+  };
+
+  return (
+    <>
+      <Helmet>
+        <title>Change Account Email</title>
+        <meta
+          name="Change account email"
```

Comment:
Oh this isnt how meta tags work, the name is for the label of the meta tag and it has to be one of a few specific strings
```suggestion
          name="description"
```